tox.ini

```diff
@@ -1,12 +1,9 @@
 [tox]
-envlist = {py34, py35, py36, py37}
+envlist = {py34, py35, py36}
 skipdist = True
 
 [testenv]
 whitelist_externals = make
-deps =
-    flake8
-    coverage
-    pytest==3.2.5
 commands =
-    make coverage ARGS='-x -vv --disable-pytest-warnings'
+    pip install -U --editable ".[dev]"
```

Comment:
We could also use this to remove this line completely:

https://tox.readthedocs.io/en/latest/config.html#conf-extras

Reply:
Thanks for the suggestion! I'll include both that option and `usedevelop`, as discussed.